_docs/principles/accessibility.md

```diff
@@ -198,61 +299,67 @@ Helix minimum spacing standards require: 
 
 <div class="hxRow"  markdown="1">
 
-{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-12-md hxSpan-12-lg" %}
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
 
-### Provide easily identifiable feedback
+### Don't use tables for layout
 
-Provide feedback for interactions, such as confirming form submission, alerting the user when something goes wrong, or notifying the user of changes on the page. Ensure instructions are easy to identify and provide suggestions to help fix issues whenever errors are returned. Prominently present any important feedback that requires user action.
+Do not use tables for formatting content or for anything other than presenting data. Because a screen reader reads every row and column of a table, including secondary text or data in additional nested rows can make interpreting the contents of a table difficult for the user.
```

Comment:
for tabular data

Reply:
the 2nd sentence is grammatically confusing can you restructure or brea into 2 sentences?

Reply:
Done.